lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -45,5 +46,47 @@ __PACKAGE__->has_many(properties => 'OpenQA::Schema::Result::WorkerProperties',
 # TODO
 # INSERT INTO workers (id, t_created) VALUES(0, datetime('now'));
 
+sub seen(;$) {
```

Comment:
Are we going to continue to use function args prototyping even on methods? I think Perl::Critic complains on this (even on non-methods).